__tests__/app.test.js

```diff
@@ -46,3 +46,39 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET 200 Responds with an article object containing the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
```

Comment:
I'd suggest having a look at the jest matchers .toMatchObject and .objectContaining for a simpler way to assert against objects (and make it easier to read)